### PR TITLE
Version control section

### PR DESCRIPTION
**Pull request by chartgerink**
Addresses #18 

# Reviewer notes

* More in-depth manual version control will be added in a Git section.

# New topics

These are topics not previously mentioned in the content or issues and that need to be added in the issues. Please use checkmarks so maintainers can keep track of which need to be added.

- [x] Metadata
- [x] Git
- [x] File naming
- [x] Project structuring

# External reviewers

Feel free to review if you want, feel free to not do so too. Much love to reviewers for their willingness to take part 😍 

@jibe-b
@amscheel
@emaassen

### CODE REVIEW

File: content/version-control.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+![](../assets/figs/version-control-diff.png)
+**Example of track changes in manual version control.**
+
+Automatic version control is easy to use but can provide excessive information because each change is stored with a timestamp. It does allow you to declutter your project folder considerably. Manual version control provides more control to the user and a more informative logbook, plus the ability to track changes within a file over time --- but requires a bit more training to use.
+
+# Exercises
+
+* Look at one of your current projects: Are there multiple versions of a similar document (e.g., manuscript) stored in the same folder?
+* Evaluate what kind of version control fits your needs better: Automatic (no logbook) or manual (with logbook)? Note: Consider whether your institution (if any) requires you to provide an account of published results.
```
Comment (rainsworth) [inline content/version-control.md]: Some journals also require an account of author contributions, which is conveniently tracked with version control.

File: content/version-control.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Manual version control systems provide the ability for the user to create snapshots of files over time and add "commit" messages to create a logbook of changes. The [Metadata](metadata.md) produced in such a logbook is valuable when retracing steps in the research process when you've forgotten when you did what. If manual version control is used with clear-text files (files you can open in notepad without problems; e.g., scripts and Markdown files), it also provides the possibility to do track changes between any two versions you specify. One way of doing manual version control is [Git](git.md).
+
+![](../assets/figs/version-control-diff.png)
+**Example of track changes in manual version control.**
```
Comment (rainsworth) [inline content/version-control.md]: track -> tracked?

**Review by rainsworth — APPROVED** [review]

File: content/version-control.md (hunk 2 above)
Comment (chartgerink) [reply content/version-control.md]: Yes! 🎢

File: content/version-control.md (hunk 1 above)
Comment (chartgerink) [reply content/version-control.md]: Updated 🌲

**Pull request merged**